ODataConnectedService/src/app.config

```diff
@@ -8,4 +8,4 @@
       </dependentAssembly>
     </assemblyBinding>
   </runtime>
-</configuration>
\ No newline at end of file
+	<startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6" /></startup></configuration>
```

Comment:
This may have been modified by Visual Studio in conjunction with the NET Framework version update. If we decide v4.6 is the way to go, feel free to ignore this comment. Otherwise, if we revert to v4.5, we'll have to change this too.